internal/cluster/cluster.go

```diff
@@ -787,10 +788,29 @@ func (m *Manager) DeleteClusterFiles(ctx context.Context, c *common.Cluster, obj
 	return nil
 }
 
-func (m Manager) PermanentClustersDeletion(ctx context.Context, olderThen strfmt.DateTime, objectHandler s3wrapper.API) error {
+func (m Manager) DeregisterInactiveCluster(ctx context.Context, inactiveSince strfmt.DateTime) error {
+	log := logutil.FromContext(ctx, m.log)
+
+	var clusters []*common.Cluster
+	db := m.db.Unscoped()
+	if reply := db.Where("updated_at < ?", inactiveSince).Find(&clusters); reply.Error != nil {
+		return reply.Error
+	}
+	for _, c := range clusters {
+		m.eventsHandler.AddEvent(ctx, *c.ID, nil, models.EventSeverityInfo,
```

Comment:
Need to add a log as well when looking at the logs i see only 
```
Deregistering all clusters that were inactive since
```

Reply:
true